src/test/java/com/zotov/edu/passportofficerestservice/PassportOfficeRestServiceApplicationTests.java

```diff
@@ -2,8 +2,10 @@
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
 
 @SpringBootTest
+@ActiveProfiles("test")
 class PassportOfficeRestServiceApplicationTests {
```

Comment:
There's no more sense in this test since we've got integration tests, so let's just drop it?

Reply:
The test class has been deleted